src/_includes/macros/activity-snippets.html

```diff
@@ -13,7 +13,7 @@
 
    activity_types:    An array of activity types.
 
-                      Possible values: posts, op-ed, press release,
+                      Possible values: posts, op-ed, newsroom, press release,
```

Comment:
Should these changes be in here? Unclear if this is a fix due to an issue discovered during testing or somehow got in from other work.